pkg/operator/controller/garden/care/health_test.go

```diff
@@ -672,7 +673,7 @@ var _ = Describe("Garden health", func() {
 				)
 				Expect(len(updatedConditions)).ToNot(BeZero())
 				Expect(updatedConditions).To(ContainElements(
-					beConditionWithStatusReasonAndMessage(gardencorev1beta1.ConditionFalse, "DeploymentUnhealthy", "Deployment \"kube-state-metrics\" is unhealthy: condition \"Available\" is missing"),
+					beConditionWithStatusReasonAndMessage(gardencorev1beta1.ConditionFalse, "DeploymentUnhealthy", "Deployment \"gardener-metrics-exporter\" is unhealthy: condition \"Available\" is missing"),
```

Comment:
Rather than replacing the previous assertion the new one should be added.

Reply:
Error return only one deployment at a time.